packages/kbn-reporting/export_types/printable_pdf_v2/printable_pdf_v2.ts

```diff
@@ -79,16 +108,16 @@ export class PdfExportType extends ExportType<JobParamsPDFV2, TaskPayloadPDFV2>
     let apmGeneratePdf: { end: () => void } | null | undefined;
     const { encryptionKey } = this.config;
 
-    const process$: Rx.Observable<TaskRunResult> = Rx.of(1).pipe(
-      mergeMap(() => decryptJobHeaders(encryptionKey, payload.headers, jobLogger)),
+    const process$: Observable<TaskRunResult> = of(1).pipe(
+      mergeMap(async () => await decryptJobHeaders(encryptionKey, payload.headers, jobLogger)),
```

Comment:
nit, for code style:
```suggestion
      mergeMap(() => decryptJobHeaders(encryptionKey, payload.headers, jobLogger)),
```